package.json

```diff
@@ -4,7 +4,7 @@
   "version": "5.25.0",
   "description": "Utilities to help when developing terra modules.",
   "engines": {
-    "node": ">=10 <13"
+    "node": ">=12 <15"
```

Comment:
Should this change be made in [package.json](https://github.com/cerner/terra-toolkit/pull/674/files#diff-72c2847b50d4152bd910ef25c33dafbd31a04d507329a333036e6226e5e8ef71) in terra-functional-testing instead of here?